server/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint consistent-return:0 */
+
+const express = require('express');
+const logger = require('./logger');
+const ngrok = require('ngrok');
+
+const frontend = require('./middlewares/frontendMiddleware');
+const isDev = process.env.NODE_ENV !== 'production';
+
+const app = express();
+
+// If you are developing more than just front-end, like POST form
+// handling or API, add your custom middleware here
+// app.use('/api', myApi);
+
+// Initialize frontend middleware that will server your JS app
+const webpackConfig = isDev
+  ? require('../internals/webpack/webpack.dev.babel')
+  : require('../internals/webpack/webpack.prod.babel');
+
+app.use(frontend(webpackConfig));
```

Comment:
This API is kind of weird, why do we pass the config to our middleware – shouldn't we pass that to our application directly? (From a user perspective)